src/schema/sources.ts

```diff
@@ -561,6 +561,7 @@ export const resolvers: IResolvers<any, Context> = {
             .andWhere(`${builder.alias}."sourceId" = :source`, {
               source: args.sourceId,
             })
+            .addOrderBy(`${builder.alias}."role"`, 'DESC')
```

Comment:
Should we add select commands to manually set the value for each role to get the desired order? Something like `IF role = 'member' THEN 1 ELSE 10` (since we don't have a table in DB to have the ordering using a column to reference).

Not sure if having additional order by criterion will affect the pagination (first time seeing it) since it is currently based on a Date property. I think it should not, but just for visibility cc @idoshamun

Reply:
I think for now it's fine. But we should definitely keep it in mind once we introduce more roles